main.ts

```diff
@@ -16,7 +16,7 @@ export interface AgileTaskNotesSettings {
 const DEFAULT_SETTINGS: AgileTaskNotesSettings = {
   selectedTfsClient: 'AzureDevops',
   targetFolder: '',
-  noteTemplate: '# {{TASK_TITLE}}\n#{{TASK_TYPE}}\n\nLink: {{TASK_LINK}}\n\n#todo:\n- [ ] Create todo list\n- [ ] \n\n## Notes:\n',
+  noteTemplate: '# {{TASK_TITLE}}\n#{{TASK_TYPE}}\n\nid: {{TASK_ID}}\n\nstate: {{TASK_STATE}}\n\nAssignedTo: {{TASK_ASSIGNEDTO}}\n\nLink: {{TASK_LINK}}\n\n#todo:\n- [ ] Create todo list\n- [ ] \n\n## Notes:\n',
```

Comment:
I like the state of the task being in the note but I don't really see the point of the id and assignedTo fields. The id is already present in the heading of the filename but I guess it could be useful to some. Currently, the assignedTo field would always display your username since the tasks are filtered for only your tasks, which makes that field redundant.

Reply:
Using this plugin, I get not only my tasks but from all the members of my team as well. And I want to use assignedTo to select only my tasks.

Reply:
Odd, I believe I coded it to filter out everyone else's tasks. Did you change the code on your end for that?

Reply:
No, I did not.
I assumed it is a wanted feature to get all the tasks from team members. Indeed, a team leader may want that to organize its team work. 
Perhaps, you can offer in the config an option to get only my tasks or the team tasks.